fastjsonschema should also be in test group

```
=========================================================================== ERRORS ============================================================================
_______________________________________________________ ERROR collecting tests/test_histogram_schema.py _______________________________________________________
ImportError while importing test module '/var/tmp/portage/dev-python/uhi-0.5.0/work/uhi-0.5.0/tests/test_histogram_schema.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.13/site-packages/_pytest/python.py:493: in importtestmodule
    mod = import_path(
/usr/lib/python3.13/site-packages/_pytest/pathlib.py:582: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1387: in _gcd_import
    ???
<frozen importlib._bootstrap>:1360: in _find_and_load
    ???
<frozen importlib._bootstrap>:1331: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:935: in _load_unlocked
    ???
/usr/lib/python3.13/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
    exec(co, module.__dict__)
tests/test_histogram_schema.py:5: in <module>
    import uhi.schema
../uhi-0.5.0-python3_13/install/usr/lib/python3.13/site-packages/uhi/schema.py:7: in <module>
    import fastjsonschema
E   ModuleNotFoundError: No module named 'fastjsonschema'
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -58,6 +58,8 @@ docs = [
 test = [
   "pytest>=6",
   "boost-histogram>=1.0",
+  "fastjsonschema",
+  "importlib-resources; python_version<'3.9'",
 ]
 
 

